Formula/qcli.rb

```diff
@@ -0,0 +1,39 @@
+class Qcli < Formula
+  desc "report audiovisual metrics via libavfilter"
+  homepage "https://bavc.org/preserve-media/preservation-tools"
+  url "https://github.com/bavc/qctools/archive/v0.8.tar.gz"
+  sha256 "5362dc8325aeb37e0742a5e5df7b831e7fe82a7b06c72c50463a43a7ad0b56bc"
+  head "https://github.com/bavc/qctools.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "qwt"
+  depends_on "qt"
+  depends_on "ffmpeg"
+
+  def install
+    ENV["QCTOOLS_USE_BREW"]="true"
+    path = ENV["PATH"]
+    ENV["PATH"] = "#{path}:#{HOMEBREW_PREFIX}/bin"
```

Comment:
Sorry, this may have been left in from testing. I removed the 2 path setting lines and it still seems to build okay.